src/api/authorityApi.ts

```diff
@@ -6,7 +6,7 @@ import { apiRequest } from './apiRequest';
 import { getIdToken } from './userApi';
 
 export enum AuthorityApiPaths {
-  PERSON = '/person',
+  PERSON = '/person-np1827', // TODO
```

Comment:
Backend må rulle ut først: https://unit.atlassian.net/browse/NP-1827
```suggestion
  PERSON = '/person',
```